pkgs/development/python-modules/into-dbus-python/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, dbus-signature-pyparsing
+, dbus-python
+, hypothesis
+, hs-dbus-signature
+}:
+
+buildPythonPackage rec {
+  pname = "into-dbus-python";
+  version = "0.08";
+
+  src = fetchFromGitHub {
+    owner = "stratis-storage";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-Z8e6oAvRMIisMjG4HcS5jSH1znGVc7pGpMITo5fXYVs=";
+  };
+
+  propagatedBuildInputs = [ dbus-signature-pyparsing dbus-python ];
+  checkInputs = [ hypothesis hs-dbus-signature ];
+
```

Comment:
It would be good to enforce testing by adding `ceckInputs = [ pytestCheckHook ]` and `pythonImprotsCheck = [ "..." ];` if possible